src/Collection/CollectionTrait.php

```diff
@@ -507,9 +521,9 @@ public function append($items)
     {
         $list = new AppendIterator();
         $list->append($this->unwrap());
-        $list->append((new Collection($items))->unwrap());
+        $list->append(($this->newCollection($items))->unwrap());
```

Comment:
Removing the extra parenthesis around `$this->newCollection($items)` should make PHP 5.6 happy.

Reply:
Somehow I did not receive updates on this pull request. I have removed the double parenthesis.